src/coreclr/vm/codeman.cpp

```diff
@@ -1516,9 +1516,12 @@ void EEJitManager::SetCpuInfo()
                         CPUCompileFlags.Set(InstructionSet_POPCNT);
```

Comment:
There is a long stale comment above "  // We will set the following flags:". It duplicates logic in this method in plain English and it is outdated (it was not updated for some of the edits of this logic).

Delete the comment? It does not seem valuable.

Reply:
Removed.